pkgs/applications/version-management/git-and-tools/git-remote-hg/default.nix

```diff
@@ -1,33 +1,32 @@
-{ stdenv, fetchgit, mercurial, makeWrapper,
+{ stdenv, fetchFromGitHub, mercurial, makeWrapper,
   asciidoc, xmlto, docbook_xsl, docbook_xml_dtd_45, libxslt, libxml2
 }:
 
 stdenv.mkDerivation rec {
-  rev = "e716a9e1a9e460a45663694ba4e9e8894a8452b2";
-  version = "0.2-${rev}";
+  version = "1.0.0";
   name = "git-remote-hg-${version}";
 
-  src = fetchgit {
-    inherit rev;
-    url = "git://github.com/fingolfin/git-remote-hg.git";
-    sha256 = "0cmlfdxfabrs3x10mfjfap8wz67s8xk2pjn2wlcj9k2v84gji60m";
+  src = fetchFromGitHub {
+    owner = "mnauw";
+    repo = "git-remote-hg";
+    rev = "765f9ae2875a2ef4439470fb4cf4adefdda2ed67";
+    sha256 = "0anl054zdi5rg5m4bm1n763kbdjkpdws3c89c8w8m5gq1ifsbd4d";
   };
 
-  buildInputs = [ mercurial.python mercurial makeWrapper
-    asciidoc xmlto docbook_xsl docbook_xml_dtd_45 libxslt libxml2
-  ];
+  nativeBuildInputs = [ makeWrapper asciidoc xmlto docbook_xsl docbook_xml_dtd_45 libxslt libxml2 ];
+  propagatedBuildInputs = [ mercurial ];
 
   doCheck = false;
 
   installFlags = "HOME=\${out} install-doc";
 
   postInstall = ''
     wrapProgram $out/bin/git-remote-hg \
-      --prefix PYTHONPATH : "$(echo ${mercurial}/lib/python*/site-packages):$(echo ${mercurial.python}/lib/python*/site-packages)${stdenv.lib.concatMapStrings (x: ":$(echo ${x}/lib/python*/site-packages)") mercurial.pythonPackages or []}"
+      --set PYTHONPATH "$(echo ${mercurial}/lib/python*/site-packages):$(echo ${mercurial.python}/lib/python*/site-packages)${stdenv.lib.concatMapStrings (x: ":$(echo ${x}/lib/python*/site-packages)") mercurial.pythonPackages or []}"
   '';
 
   meta = with stdenv.lib; {
-    homepage = https://github.com/felipec/git-remote-hg;
+    homepage = https://github.com/mnauw/git-remote-hg;
     description = "Semi-official Mercurial bridge from Git project, once installed, it allows you to clone, fetch and push to and from Mercurial repositories as if they were Git ones";
```

Comment:
Can we shorten this?
It's pretty long for `description` which is meant to be short and to the point.